app/src/main/res/layout/fragment_navigation_overlay.xml

```diff
@@ -46,6 +46,17 @@
             app:layout_constraintStart_toStartOf="parent"
             android:background="@color/photonGrey70"/>
 
+    <View
```

Comment:
Add a comment that this view actually dismiss the overlay by allowing tap to fall through to the dismiss overlay view.